controllers/inferenceservice_controller.go

```diff
@@ -152,7 +150,22 @@ func (r *OpenshiftInferenceServiceReconciler) SetupWithManager(mgr ctrl.Manager)
 				}
 				return reconcileRequests
 			}))
-	err := builder.Complete(r)
+
+	// check if kserve is enabled, otherwise don't require Authorino.
```

Comment:
will this work without watching for changes to the DSC CR? If the management state of Kserve is changed after the initial install, odh model controller might have an incorrect assumption about the state of kserve

Reply:
also iirc, authorino integration is `opt-in`, right? i.e our components should work when Kserve is enabled but Authorino might not be installed? 
cc @israel-hdez

Reply:
Authorino is not opt-in. However, in `RawDeployment` mode (i.e. when KServe is installed without Service Mesh nor Knative), Authorino will be missing.

These changes are good, but does not seem to cover `RawDeployment` mode.

> will this work without watching for changes to the DSC CR?

Being this manager setup, I don't think we can re-setup the controller once manager loop has started. So, changing the managementState of KServe or ServiceMesh/deploymentMode would require a restart of all replicas of odh-model-controller.

Reply:
The RawDeployment might be tricky in this case, the setup will happens before any reconcile, and usually users will install the ISVC after creating the DSC.
With that said, users might be requiring to have the isvc pre-defined already.

For RawDeployment, I guess we can do it by adding flags through annotations or config-map configuration to enable or disable authorino integration.

Reply:
okays, updated.

Reply:
test steps updated as well.